code/chart.js

```diff
@@ -2,3 +2,54 @@
 const ctx = document.getElementById('chart').getContext('2d')
 
 //"Draw" the chart here 👇
+
+const callingChart = (amountOfRepos) => {
+
+
+const labels = [
+    'Projects done',
+    'Projects left to do'
+  ];
+
+  const data = {
+    labels: labels,
+    datasets: [{
+      label: 'Technigo Projects',
+      backgroundColor: ['rgb(82, 183, 136)', 'rgb(0, 108, 117)'],
+      borderColor: 'rgb(255, 99, 132)',
+      data: [amountOfRepos, 19-amountOfRepos],
+    }]
+  };
+
+  const config = {
+    type: 'bar',
```

Comment:
I really like that you used a bar chart to showcase your projects and your progression. Also that you put it on top of the page! The chart is also responsive which is great :)